### DISCUSSION TITLE
Use Deeplinks on Capacitor without Framework

### DISCUSSION THREAD

**Discussion by BerkeAras** [ionic-team/capacitor#3456] [Q&A] [answered]
Hello 😄 

How can I use Deeplinks without using Frameworks like Angular, React or Vue? The documentation just shows it for frameworks ([https://capacitorjs.com/docs/guides/deep-links](https://capacitorjs.com/docs/guides/deep-links))

**Comment by ikeith** [comment, marked as answer]
Most of the documentation still applies. You need to add a listener for the `appUrlOpen` event, which will pass in the URL that the user followed. The examples in the docs pass that slug to the respective framework's router but your app can do something else with it if you're handling internal navigation differently.
